node/core/provisioner/src/metrics.rs

```diff
@@ -34,6 +34,11 @@ struct MetricsInner {
 pub struct Metrics(Option<MetricsInner>);
 
 impl Metrics {
+	/// Creates new dummy `Metrics` instance. Used for testing only.
+	pub fn new_dummy() -> Metrics {
+		Metrics(None)
+	}
+
```

Comment:
```suggestion
``

or  limit it `#[cfg(test)]`

Reply:
Let's go with #[cfg(test)] otherwise we have to make `MetricsInner` pub(crate) too.